akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -143,9 +151,11 @@ private[remote] class SystemMessageDelivery(
       private val nackCallback = getAsyncCallback[Nack] { reply ⇒
         if (reply.seqNo <= seqNo) {
           ack(reply.seqNo)
-          if (reply.seqNo > resendingFromSeqNo)
-            resending = unacknowledged.clone()
-          tryResend()
```

Comment:
this was the actual change, rest is more logging and timing adjustments of tests